src/main/kotlin/com/linker/linkerapi/notification/service/DiscordNotificationService.kt

```diff
@@ -0,0 +1,62 @@
+package com.linker.linkerapi.notification.service
+
+import com.linker.linkerapi.notification.api.DiscordWebhookApi
+import com.linker.linkerapi.notification.dto.DiscordEmbed
+import com.linker.linkerapi.notification.dto.DiscordEmbedField
+import com.linker.linkerapi.notification.dto.DiscordWebhookDto
+import com.linker.linkerapi.rental.entity.Rental
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+
+@Service
+class DiscordNotificationService(
+    private val discordWebhookApi: DiscordWebhookApi
+) {
+    @Value("\${notification.discord.webhook-url}")
+    private lateinit var webhookUrl: String
```

Comment:
```suggestion
class DiscordNotificationService(
    @Value("\${notification.discord.webhook-url}")
    private val webhookUrl: String
    private val discordWebhookApi: DiscordWebhookApi
) {
```

생성자에서도 di 받을 수 있습니다.